src/BloomBrowserUI/bookLayout/basePage.less

```diff
@@ -257,17 +257,27 @@ preview.css.
 //currently, in edit mode every page looks like page 1 to css, so left-vs-right margins are just confusing
 //so we only turn them on when making PDF
 .publishMode :not(.outsideFrontCover):not(.outsideBackCover){
-  &.bloom-page:nth-of-type(odd) .marginBox  {
-	  left: @MarginOuter;
-  }
-  &.bloom-page:nth-of-type(even) .marginBox  {
-	  left: @MarginInner;
-  }
+	&.bloom-page:nth-of-type(odd){
```

Comment:
OK. I was able to simplify these rules as you mention, but I needed to make one rule in wallCalendar "important", since it applies in both Edit and Publish mode and is specific to the wall calendar.